pkg/storage/test/tuples.go

```diff
@@ -785,7 +785,7 @@ func TupleWritingAndReadingTest(t *testing.T, datastore storage.OpenFGADatastore
 		require.NoError(t, err)
 		require.Nil(t, tp.GetKey().GetCondition())
 
-		changes, _, err := datastore.ReadChanges(ctx, storeID, "", storage.PaginationOptions{}, 0)
+		changes, _, err := datastore.ReadChanges(ctx, storeID, "", storage.NewPaginationOptions(0, ""), 0)
```

Comment:
Can you please also:

1. Change all usages of `storage.NewPaginationOptions(0,...)`?
2. Fix `NewPaginationOptions`?

```go
// NewPaginationOptions creates a new [PaginationOptions] instance
// with a specified page size and continuation token. If the input page size is empty,
// it uses DefaultPageSize.
func NewPaginationOptions(ps int32, contToken string) PaginationOptions {
	pageSize := DefaultPageSize
	if ps != 0 {
		pageSize = int(ps)
	}
```

it should be 


```go
// NewPaginationOptions creates a new [PaginationOptions] instance
// with a specified page size and continuation token. If the input page size is less than 1,
// it uses DefaultPageSize.
func NewPaginationOptions(ps int32, contToken string) PaginationOptions {
	pageSize := DefaultPageSize
	if ps >= 1 0 {
		pageSize = int(ps)
	}
```

3. Finally, update the Godoc here

```
// PaginationOptions holds the settings for pagination in data retrieval operations. It defines
// the number of items to be included on each page (PageSize) and a marker from where to start
// the page (From).
type PaginationOptions struct {
```

to

```
// PaginationOptions should not be instantiated directly. Use NewPaginationOptions.
type PaginationOptions struct {
```